lib/ClangImporter/ImportType.cpp

```diff
@@ -1866,6 +1885,9 @@ class GetSendableType :
     // These types are produced during bridging and have conditional
     // conformances to Sendable depending on their generic parameters, so we
     // want to make their generic parameters `Sendable`.
+    // FIXME: `Unmanaged` ought to be here too, but `AnyObject & Sendable`
+    //        doesn't satisfy the generic parameter's `AnyObject` requirement.
+    //        (rdar://90946615)
```

Comment:
Not a big deal, but I prefer not to have Radar numbers in source code, because they're not useful outside of Apple. The FIXME text already suffices to express the issue.